vnf.scad

```diff
@@ -481,7 +481,7 @@ function _triangulate_planar_convex_polygons(polys) =
         tris = [for (poly=polys) if (len(poly)==3) poly],
         bigs = [for (poly=polys) if (len(poly)>3) poly],
         newtris = [for (poly=bigs) select(poly,-2,0)],
-        newbigs = [for (poly=bigs) select(poly,0,-2)],
```

Comment:
Maybe leave this one as it is because it shows more directly how newbigs and newtris relate?